lib/tasks/deploy.rake

```diff
@@ -73,6 +71,21 @@ namespace :deploy do
     puts "Build URL: #{response['build_url']}"
   end
 
+  def deploy_envs
+    return DEPLOY_ENVS unless ENV.key?('DEPLOY_ENVS')
+    # e.g., DEPLOY_ENVS=master:production,staging:staging,staging:preview
```

Comment:
I suppose you can always accidentally misconfigure the ENV, though, even with your strategy.  I just went with the current format because it makes fewer assumptions about the server environment (since this is still technically OSS).